docs/ref-manual/cvl/invariants.md

```diff
@@ -154,15 +155,17 @@ commands are also possible.
 
 Preserved blocks are listed after the invariant expression (and after the filter
 block, if any), inside a set of curly braces (`{ ... }`).  Each preserved block
-consists of the keyword `preserved` followed by an optional method signature,
-an optional `with` declaration, and finally the block of commands to execute.
+consists of the keyword `preserved` followed by an optional method signature 
+(or `fallback()`), an optional `with` declaration, and finally the block of commands 
+to execute.
 
 If a preserved block specifies a method signature, the signature should
 match one of the contract methods, and the preserved block only applies when
-checking preservation of that contract method.  The arguments of the method are
-in scope within the preserved block.  If there is no method signature, the
-preserved is a default block that applies to all methods that don't have a
-specific preserved block.
+checking preservation of that contract method.  The fallback() preserved block
+applies only to the `fallback()` function that should be defined in the contract.
```

Comment:
if `fallback` applies to the default fallback as well, then I would say "applies to the contract's fallback function" and drop the "which should...".  Otherwise, I would probably make it clearer that it does not apply to the default fallback

Reply:
I didn't see any reference in the Solidity documentation to a "default fallback" function being created by the solc. What I understood is that in case Ether is sent to a contract without fallback (or `receive`) function, an exception is raised and the Ether is sent back to the caller.

Reply:
ok, sounds good.